src/test/loader.ts

```diff
@@ -216,7 +226,7 @@ function toReporters(reporters: 'dot' | 'line' | 'list' | 'junit' | 'json' | 'nu
 }
 
 function errorWithFile(file: string, message: string) {
-  return new Error(`${file}: ${message}`);
+  return new Error(`${path.relative(process.cwd(), file)}: ${message}`);
```

Comment:
drive-by: all these error messages had ugly absolute paths. Now they should be a lot smaller and nicer.